src/type-layout.md

```diff
@@ -1,11 +1,30 @@
 # Type Layout
 
-The layout of a type is its size, alignment, and the relative offsets of its
-fields. For enums, how the discriminant is laid out and interpreted is also part
-of type layout.
+The *layout* of a type is the combination of its:
 
-Type layout can be changed with each compilation. Instead of trying to document
-exactly what is done, we only document what is guaranteed today.
+* size
+* alignment
+* relative offsets of its fields or elements
+* relative offsets of padding
+* for enums: how the discriminant is laid out and interpreted
+
+> **Note**: Type layout is allowed to change with each compilation. Instead of
+> trying to document exactly how each type is laid out, we only document the
+> properties that are guaranteed.
+
+## Padding
+
+Types may contain *padding*: bytes that have no semantic meaning that may take
+on any bit pattern. When a type is [moved or copied], the value of padding bytes
+may be changed.
+
+If a field can be accessed, its padding is also the padding of the containing
+type.
```

Comment:
I'm having a little difficulty understanding this sentence.  Can you elaborate on what it is saying? I've always had the impression that padding is the space *between* fields, not a property of the field itself.  And I'm uncertain what "the padding of the containing type" is.